docs/docs/guides/web3_config/index.md

```diff
@@ -201,6 +202,610 @@ console.log(web3.getContextObject().config)
+```
+:::info
+The `transactionBlockTimeout` can be configured both globally and at the package level:
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.eth.transactionBlockTimeout = 50;
+
+console.log(web3.eth.getContextObject().config);
+```
+:::
+
+### transactionConfirmationBlocks
+This defines the number of blocks it requires until a transaction is considered confirmed. Different chains have varying security considerations and requirements for confirmation block numbers.
+
+The default value of `transactionConfirmationBlocks` is 24.
+
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.transactionConfirmationBlocks = 24;
+
+console.log(web3.getContextObject().config);
+```
+:::info
+The `transactionConfirmationBlocks` can be configured both globally and at the package level:
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.eth.transactionConfirmationBlocks = 24;
+
+console.log(web3.eth.getContextObject().config);
+```
+:::
+
+
+### transactionPollingInterval
+The `transactionPollingInterval` is used over HTTP connections. This option defines the number of seconds between Web3 calls for a receipt which confirms that a transaction was mined by the network. Modifying this value can reduce the wait time for confirmations or decrease the number of network requests. Setting the `transactionPollingInterval` would also set `transactionReceiptPollingInterval` and `transactionConfirmationPollingInterval` to the same value. 
+
+The default value of `transactionPollingInterval` is 1000 ms.
+
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.transactionPollingInterval = 1000; // 1000 ms = 1 s
+
+console.log(web3.getContextObject().config);
+/* ↳
+  ...
+  transactionPollingInterval: 1000,
+  transactionPollingTimeout: 750000,
+  transactionReceiptPollingInterval: 1000,
+  transactionSendTimeout: 750000,
+  transactionConfirmationPollingInterval: 1000,
+  ...
+*/
+```
+:::info
+The `transactionPollingInterval` can be configured both globally and at the package level:
+```ts
+import { Web3 } from 'web3';
+
+const web3 = new Web3('http://127.0.0.1:7545');
+
+web3.eth.transactionPollingInterval = 1000; // 1000 ms = 1 s
+
+console.log(web3.eth.getContextObject().config);
+/* ↳
+  ...
+  transactionPollingInterval: 1000,
+  transactionPollingTimeout: 750000,
+  transactionReceiptPollingInterval: 1000,
+  transactionSendTimeout: 750000,
+  transactionConfirmationPollingInterval: 1000,
+  ...
+*/
+```
+:::
+
+### transactionPollingTimeout
+The `transactionPollingTimeout` is used over HTTP connections. This option defines the number of seconds Web3 will wait for a receipt which confirms that a transaction was mined by the network. It can be set based on the average transaction confirmation time on the network. Note: If this method times out, the transaction may still be pending. 
```

Comment:
Does this seem more accurate?

```suggestion
The `transactionPollingTimeout` is used over HTTP connections. This option defines the number of seconds Web3 will wait for a receipt which confirms that a transaction was mined by the network. It can be set based on the average transaction confirmation time on the network. Note: If the `transactionPollingTimeout` is exceeded, the transaction may still be pending. 
```

Reply:
Definitely, it's better